vhdbuilder/packer/vhd-scanning.sh

```diff
@@ -222,6 +222,13 @@ isFlatcar() {
         return 0
     fi
 }
+isAzureLinuxOSGuard() {
+    local os="$1"
+
+    if [ "$os" = "AzureLinuxOSGuard" ]; then
+        return 0
+    fi
```

Comment:
we need a `return 1` at the end here

Reply:
Ack, also adding it to the isFlatcar function above.